sophie_pub.snakefile

```diff
@@ -0,0 +1,233 @@
+# import all of the sophie libraries.
+# I chose to run all of the python sophie code via the "run" directive, which doesn't allow for the use of an environment.
```

Comment:
```suggestion
# I chose to run all of the python sophie code via the "run" directive, which doesn't allow for the use of an environment.
# I made this choice because it's easier to keep track of everything that is being run, and is more similar to the notebooks that Alex had originally created for running sophie.
```